lib/exportJob.js

```diff
@@ -19,17 +19,70 @@ const logger = debug('electronpdf:')
 const wargs = require('./args')
 
 // CONSTANTS
+/** Used to calculate browser dimensions based on PDF size */
 const HTML_DPI = 96
+/** Interval for which to check for hung windows */
```

Comment:
I think it would be helpful to list the units, and what 30,000 translates into  Same comment for HUNG_WINDOW_THRESHOLD.